docker-compose.yaml

```diff
@@ -11,3 +11,4 @@ services:
     privileged: true
     volumes:
       - /dev:/dev
```

Comment:
privileged: true

might already give /dev access. test this and remove the mount if it does.